rastersample.ts

```diff
@@ -0,0 +1,39 @@
+import { Style } from 'index';
+
+const sampleRasterStyle: Style = {
+  name: 'Sample Raster Style',
+  rules: [
+    {
+      name: 'Very old Peter',
```

Comment:
Are you really VERY OLD if you are beyond the age of 51? :wink:

Reply:
Whats my age again?